src/registry/domain/repository.js

```diff
@@ -26,12 +27,7 @@ module.exports = function(conf){
   var coreTemplates = ['oc-template-jade', 'oc-template-handlebars'];
   var templates = _.union(coreTemplates, conf.templates)
     .map(function(template){
-      var info;
-      try {
-        info = require(template).getInfo();
-      } catch (err) {
-        throw new Error(format(strings.errors.registry.TEMPLATE_NOT_FOUND, template));
-      }
```

Comment:
Let's re-add a try catch here for a template that, in case is not valid, may throw as getInfo doesn't exist or is not a function. Error wouldn't be "template not found" anymore but more of a "template not valid" kind?